great_expectations/rule_based_profiler/rule/rule.py

```diff
@@ -44,7 +44,7 @@ def __init__(
 
         self._parameters = {}
 
-    def generate(
+    def run(
```

Comment:
Should we reflect this naming change in the docstring at all?

Reply:
@cdkini I did not find any mention of `generate()` in the existing docstrings, so felt like we can just rename.